flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/TableLoader.java

```diff
@@ -38,6 +38,8 @@ public interface TableLoader extends Closeable, Serializable, Cloneable {
 
   void open();
 
+  boolean isOpen();
```

Comment:
is this necessary?

Reply:
I guess we can revisit this when we moved back to supplier model.

Reply:
I'm still using this API in the new supplier, let me know if you'd still like me to remove it.